docs/system/wind_turbine.md

```diff
@@ -1,7 +1,147 @@
+
+| Advantages                            | Drawbacks                                     |
+|---------------------------------------|-----------------------------------------------|
+| High Power coefficient                | Yaw system to follow wind needed              |
+| No pulsating torque on drive train    | High noise generation due to high tip speed   |
+| Well understood and developed         | Complex blade design and production           |
+
+#### VAWT
+
+Vertical axis wind turbines have a power coefficient of $c_p \approx 30\%-40\%$ and are not often found in commercial setups. They have however some advantages that makes them suitable for a range of applications. Namely their low noise profile and unusal appearance give them high acceptance in urban areas. They also do not require a yaw system and are better suited for turbulent areas. Most common designs are the Darriues turbine, the helical bladed turbine and the straight-bladed Darrieus turbines (see [Wikipedia](https://en.wikipedia.org/wiki/Darrieus_wind_turbine) for more details)
+
+| Advantages                            | Drawbacks                                     |
+|---------------------------------------|-----------------------------------------------|
+| No yaw mechanism needed               | Medium Power coefficient                      |
+| Low noise profile                     | High normal forces on axis                    |
+| Simple blade geometry                 | Oscillating torque on generator               |
+
+## Generators
+
+Electric generators can be classified into radial flux machines and axial flux machines. The main difference is the flow direction of magnetic flux, where it is either parallel or radial to the rotaional axis. The common synchronous and asynchronous (induction) machines belong to the radial flux machines and are mainly used in large turbines. One of the main challenges for larger wind turbines is the connection to the grid. While DC-Sources can be connected to the grid when their voltage is equal to the grid voltage, AC-Sources have to be in synch with the grid's voltage frequency and amplitude [4]. For small scale, especially off-grid turbines, axial flux generators are more common as they can be fairly easily constructed without high level machinery, especially for low-rpm usecases.
```

Comment:
Slight reordering of the 3rd sentence as previously one could understand that synchronous or asynchronous machines are always radial flux, which is not the case.
The sentence regarding DC and AC sources is not completely clear to me. Are we talking about DC or AC grids? If you meant also DC grid, maybe use the term "DC system as used in many off-grid applications" instead of DC grid because DC grids are not really common (yet).
```suggestion
Electric generators can be classified into radial flux machines and axial flux machines. The main difference is the flow direction of magnetic flux, where it is either parallel or radial to the rotational axis. The commonly used synchronous and asynchronous (induction) machines used in large turbines belong to the category of radial flux machines. One of the main challenges for larger wind turbines is the connection to the grid. While DC sources can be connected to the grid when their voltage is equal to the grid voltage, AC sources have to be in sync with the grid's voltage frequency and amplitude [4]. For small scale, especially off-grid turbines, axial flux generators are more common as they can be fairly easily constructed without high level machinery, especially for low-rpm usecases.
```